test/test_transforms_v2_refactored.py

```diff
@@ -1634,3 +1637,143 @@ def test_transform_negative_degrees_error(self):
     def test_transform_unknown_fill_error(self):
         with pytest.raises(TypeError, match="Got inappropriate fill arg"):
             transforms.RandomAffine(degrees=0, fill="fill")
+
+
+class TestCutMixMixUp:
+    # TODO: Does it work when labels are already dirichlet-distributed? like does Compose([Mixup(), CutMix()]) work?
+    @pytest.mark.parametrize("T", [transforms.Cutmix, transforms.Mixup, "Compose"])
```

Comment:
Instead of parametrizing over the class / name here, should we maybe parametrize over the instance along with `expected_num_non_zero_labels`? Plus, we should probably also do the "reversed compose", i.e. Mixup before Cutmix.